### PR TITLE
feature/SM-6687-Update API docs for Sandbox release

### PR DESCRIPTION
**Pull request by RichardEustace**
Added a V1.25.10 - I see there is already a V1.26 (what is the approach?)

### CODE REVIEW

File: docs/api-documentation/V1.25.10/index.md [hunk 1]
```diff
@@ -0,0 +1,2778 @@
...
+  <li><a class="govuk-link" href="#technical-overview">Technical Overview</a></li>
+  <li><a class="govuk-link" href="#versioningandreleasemanagement">Versioning and Release Management</a></li>
+  <li><a class="govuk-link" href="#integration-guide">Integration guide</a></li>
+  <li><a class="govuk-link" href="#testing">Testing</a></li>
+  <li><a class="govuk-link" href="#security">Security</a></li>
+  <li><a class="govuk-link" href="#sequencing">Sequencing</a></li>
+  <li><a class="govuk-link" href="#access-and-permissions">Access and permissions</a></li>
+  <li><a class="govuk-link" href="#resource-guide">Resource Guide</a></li>
+  <li><a class="govuk-link" href="#roadmap">Roadmap</a></li>
+  <li><a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.25.9/changelog">Versions and Changes</a></li>
```
Comment (cpaul92) [inline docs/api-documentation/V1.25.10/index.md:34]:
```suggestion
  <li><a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.25.10/changelog">Versions and Changes</a></li>
```

File: docs/api-documentation/V2.10.5/index.md [hunk 2]
```diff
@@ -0,0 +1,2785 @@
...
+  <li><a class="govuk-link" href="#technical-overview">Technical Overview</a></li>
+  <li><a class="govuk-link" href="#versioningandreleasemanagement">Versioning and Release Management</a></li>
+  <li><a class="govuk-link" href="#integration-guide">Integration guide</a></li>
+  <li><a class="govuk-link" href="#testing">Testing</a></li>
+  <li><a class="govuk-link" href="#security">Security</a></li>
+  <li><a class="govuk-link" href="#sequencing">Sequencing</a></li>
+  <li><a class="govuk-link" href="#access-and-permissions">Access and permissions</a></li>
+  <li><a class="govuk-link" href="#resource-guide">Resource Guide</a></li>
+  <li><a class="govuk-link" href="#roadmap">Roadmap</a></li>
+  <li><a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.10.4/changelog">Versions and Changes</a></li>
```
Comment (cpaul92) [inline docs/api-documentation/V2.10.5/index.md:34]:
```suggestion
  <li><a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.10.5/changelog">Versions and Changes</a></li>
```

**Comment by pwright08** [comment]
>I see there is already a V1.26

@RichardEustace I don't see a v1.26 in the repo, are you sure that's not just locally for you?

See https://github.com/departmentfortransport/street-manager-docs/tree/master/docs/api-documentation

File: docs/api-documentation/index.html [hunk 3]
```diff
@@ -87,14 +87,14 @@ <h1 class="govuk-heading-m">Releases</h1>
...
+      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V2.10.5/">
+        Version 2.10.5 - Due for release to Sandbox on 04/02/2021. Due for release to Production on 08/02/2021.
       </a>
     </li>
     <!-- UPDATE ME to upcoming stable version if required  -->
     <li>
-      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.25.9/">
-        Version 1.25.9 - Due for release to Sandbox on 21/01/2021. Due for release to Production on 25/01/2021.
+      <a class="govuk-link" href="{{ site.baseurl }}/api-documentation/V1.25.10/">
+        Version 1.25.10 - Due for release to Sandbox on 04/02/2021. Due for release to Production on 08/02/2021.
```
Comment (pwright08) [inline docs/api-documentation/index.html:97]: I think we normally comment out the link to a new stable version until we actually need to use it. I.e. if next release we don't add any new stable API docs, we would just save it for the next time we do

**Review by cpaul92 — CHANGES_REQUESTED** [review]
Were there no swagger changes to the previous versions? Also, you're right with the approach of adding 1.25.10, I don't see the V1.26 you mentioned.. am I missing something?

File: docs/api-documentation/V1.25.10/index.md (hunk 1 above)
Comment (RichardEustace) [reply docs/api-documentation/V1.25.10/index.md:34]: I downloaded the Swagger from -
https://api.sandbox.manage-roadworks.service.gov.uk/v1/work/swagger.json
https://api.sandbox.manage-roadworks.service.gov.uk/v2/work/swagger.json
into relevant folder and do not see any diffs

the v1.26 is an empty local dir - ignore me

**Comment by RichardEustace** [comment]
> > I see there is already a V1.26
> 
> @RichardEustace I don't see a v1.26 in the repo, are you sure that's not just locally for you?
> 
> See https://github.com/departmentfortransport/street-manager-docs/tree/master/docs/api-documentation

the v1.26 is an empty local dir - ignore me

File: docs/api-documentation/V1.25.10/index.md (hunk 1 above)
Comment (pwright08) [reply docs/api-documentation/V1.25.10/index.md:34]: Looking at the list of stories from last sprint, I can't see any that I think should have API interface changes

Comment (RichardEustace) [reply docs/api-documentation/V1.25.10/index.md:34]: Good spot on the change log - And I added it lol!

**Review by pwright08 — APPROVED** [review]

**Review by cpaul92 — APPROVED** [review]

**Pull request merged**